model/service_mgmt/v1/service_resource.model

```diff
@@ -0,0 +1,24 @@
+/*
+Copyright (c) 2022 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Manages Managed Services.
+resource Service {
+
+  // Creates a new Managed Service
+	method Add {
```

Comment:
Sounds strange that you have a method to add a service, but no methods to list services or get a specific service. Is that on purpose?